.node-version

```diff
@@ -0,0 +1 @@
+8.1.0
```

Comment:
This file is only used by nodenv; travis-ci uses nvm. Also is v8.1 even out?

Please remove this file.